site/app/libraries/PollUtils.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+namespace app\libraries;
+
+use app\libraries\Core;
+use app\libraries\FileUtils;
+use app\libraries\response\JsonResponse;
+
+/**
+ * Class FileUtils
+ *
+ * Contains various useful functions for the polling system.
+ */
+class PollUtils {
+    /**
+     * Generates a PHP array containing the all the poll data used in the export procedure
+     * @param array $polls the array of polls to be summarized
+     */
+    public static function getPollExportData(array $polls): array {
```

Comment:
can you add a test case or two (e.g. test for empty `$polls` array, test for array with few elements) for this method?